build/Dockerfile

```diff
@@ -131,16 +134,18 @@ apt-get update
 apt-get install -y --no-install-recommends \
     ca-certificates \
     curl \
-    git \
-    jq \
-    wget \
-    xxd
+    git
 EOF
 
 # Install Foundry from downloaded pre-compiled binaries
 ARG FOUNDRY_NIGHTLY_VERSION
 RUN curl -sSL https://github.com/foundry-rs/foundry/releases/download/nightly-${FOUNDRY_NIGHTLY_VERSION}/foundry_nightly_linux_$(dpkg --print-architecture).tar.gz | \
-    tar -zx -C /usr/local/bin
+        tar -zx -C /usr/local/bin
+
+# STAGE: devnet-golang
+#
+# This stage makes GO_VERSION available locally
+FROM golang:${GO_VERSION}-bookworm as devnet-golang
```

Comment:
My bad. Changed at b90cc13d

Reply:
> Is there any advantage in creating this extra stage to simply installing Golang directly on the devnet-builder stage?

One reason is the registry's cache. It may be faster to build many times using `FROM ...` than without it.

Reply:
Sure, good point, but in that case it was just a matter of working around an issue and, considering this is an internal tool, I'd rather go with @torives's suggestion and simply install Go on it's own layer.